test/geometry_tests.cpp

```diff
@@ -20,6 +20,15 @@ void printPixels(std::vector<std::vector<int>> pixels){
 	std::cout << "\n";
 }
 
+void printPixels(std::vector<Cell> v){
```

Comment:
Add doxygen.
Pass by const ref

Reply:
Actually, you should just delete this, as well as the commented out ones below

Reply:
But if you wanted to keep some version of this, gtest supports this syntax
```
EXPECT_TRUE(...stuff) << "Additional message to print when test fails";
```

Reply:
```
std::ostream operator<<(std::ostream& os, const std::vector<Cell>& v);
```

Reply:
```
std::cout << v1 << v2;
```
is equivalent to
```
operator<<(operator<<(std::cout, v1), v2)
```

Reply:
Done